types/office-js/index.d.ts

```diff
@@ -30438,7 +30438,7 @@ declare namespace Excel {
         /**
          * Specifies if the password can be used to unlock worksheet protection.
                     This method doesn't change the worksheet protection state.
-                    If a password is inputted but no password is required to unlock worksheet protection, this method will return false.
+                    If a password is input but no password is required to unlock worksheet protection, this method will return false.
```

Comment:
The Microsoft Style Guide says to use "entered" not "input" or "inputted".

Reply:
Thanks! I'll make a note to fix this in the source. It'll reach DefinitelyTyped in a few weeks or so.